web_src/js/index.js

```diff
@@ -2240,6 +2335,17 @@ function initTemplateSearch() {
 }
 
 $(document).ready(async () => {
+  // Show issue or pr in board sidebar
+  $('.board-column').on('click', '.draggable-cards [data-url]', (e) => {
+    e.preventDefault();
+    $('#current-card-details').html('');
+    $('#current-card-details-input').html('');
+    const data = $(e.currentTarget).data();
+    $('#current-card-details').css('visibility', 'visible');
+    $('#current-card-details').show();
+    reloadIssuesActions(data);
+  });
+
```

Comment:
A lot of these changes could live in `web_src/js/features/projects`, please move them there. index.js is already huge enough 😉.

Reply:
@silverwind I did put as much code as I could in projects.js but the code I left in index.js depended on code that is in index.js